.expeditor/scripts/build_mac_component.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/local/bin/bash
+
+set -euo pipefail
+
+source .expeditor/scripts/shared.sh
+
+# Get secrets!
+eval "$(vault-util fetch-secret-env)"
```

Comment:
This is pulling/injecting our HAB_AUTH_TOKEN and AWS creds from vault

Reply:
We override `HAB_AUTH_TOKEN` immediately below, and I don't see where we're interacting with AWS. 

Could we just remove this line, then?

Reply:
I wasn't clear, sorry - for right now we're pulling the ACCEPTANCE_HAB_AUTH_TOKEN from vault (which is my acceptaance token) from vault and overriding it. When we go live with this we can skip the `export` line here because it'll already be in the env.

Reply:
Is there a reason why we're not setting that in the pipeline like the rest of the build steps? If not, it would be nice to have them all be as self-consistent as possible.

(it would also be good to add a comment that indicates which secrets in particular we're intending to pull in)

Reply:
They are actually: https://github.com/habitat-sh/habitat/blob/master/.expeditor/release_habitat.pipeline.yml#L1-L6

We only have this extra "eval" because of this bug: https://github.com/chef/ci-studio-common/issues/200

Reply:
I will totally add some comments to this though explaining the situation :)

Reply:
Great! A link to that issue will definitely help :smile: 

Thanks!